AGW-4213: Review MMA documentation (#46)
mulesoft/docs-beta: modules/ROOT/pages/migration-api-gateways-proxies.adoc

=== POM Migration

The POM file migration modifies the file to include the elements necessary for
uploading the custom proxy to Exchange.

> After the migration:

Reviewer: ```suggestion
_After_ the migration:
```

> Replace the `{orgId}` value in the `<groupId/>` and `<exchange.url/>` elements with the organization ID found in Anypoint Platform.

Reviewer: ```suggestion
* Replace the `{orgId}` value in the `<groupId/>` and `<exchange.url/>` elements with the organization ID found in Anypoint Platform.
```

[[wsdl_proxy_migration]]
=== WSDL Proxy Migration

> WSDL Proxy migration consists on migrating the attribute values for WSDL properties. In Mule 3, most values where extracted using the function defined as attribute value. In Mule 4 there is no equivalent for doing so, therefore most properties are migrated to a default value and require manual configuration from the user with values taken from the WSDL file. These are the ones starting with the `service` keyword.

Reviewer: ```suggestion
WSDL proxy migration consists of migrating the attribute values for WSDL properties. In Mule 3, most values are extracted using the function defined as attribute value. In Mule 4, there is no equivalent process, so most properties are migrated to a default value and require manual configuration with values from the WSDL file, which start with the `service` keyword.
```

> Some properties are just renamed in order to extract in a more transparent way the value, such as:

Reviewer: ```suggestion
Some properties are renamed to extract the value in a more transparent way, for example:
```

[cols="2,5",options="header"]
|===
|Mule 3                                  | Mule 4

| `![wsdl(p['wsdl.uri']).services[0].preferredPort.name]`	| `${service.port}`
| `![wsdl(p['wsdl.uri']).targetNamespace]` 					| `${service.namespace}`
| `![wsdl(p['wsdl.uri']).services[0].name]`					| `${service.name}`
|===

> However, other properties such as the following are migrated to a DataWeave expression which uses the WSDL Functions Extension dependency, provided by MMA, to extract the expected values:

Reviewer: ```suggestion
Other properties, such as the following, are migrated to a DataWeave expression that uses the WSDL Functions Extension dependency, provided by MMA, to extract the expected values:
```
